Sources/AWSSDKSwiftCore/AWSClient.swift

```diff
@@ -570,17 +606,21 @@ extension AWSClient {
 }
 
 // response validator
-extension AWSClient {
+extension AWSHTTPResponse {
```

Comment:
Request creation has been moved to `AWSRequest` and response validation has been moved to `AWSHTTPResponse`. That's a little unbalanced. Shouldn't most of the Response validation be in AWSResponse?